components/Navbar/MobileNav.tsx

```diff
@@ -0,0 +1,123 @@
+import { toast } from "../ui/use-toast";
+import { usePathname, useRouter } from "next/navigation";
+import { Sheet, SheetContent, SheetTrigger } from "../ui/sheet";
+type NavbarProps = {
+  session: Session | null;
+};
+
+const MobileNav = ({ session }: NavbarProps) => {
+  const [open, setOpen] = useState(false);
+  const router = useRouter();
+  const pathName = usePathname();
+
+  const handleSignOut = async () => {
+    const response = await logOutUser();
+
+    if (response?.status !== "success") {
+      toast({
+        variant: "destructive",
+        title: response?.message,
+      });
+      return;
+    }
+
+    router.push("/");
+    router.refresh();
+  };
+
+  const userRole = session?.user.role;
+
+  return (
+    <Sheet open={open} onOpenChange={setOpen}>
+      <SheetTrigger asChild>
+        <Button variant="outline" className="w-10 px-0 sm:hidden">
+          <Menu className="h-5 w-5" />
+          <span className="sr-only">Toggle Theme</span>
+        </Button>
+      </SheetTrigger>
+      <SheetContent side="right">
+        <MobileLink
+          onOpenChange={setOpen}
+          href="/"
+          className="flex items-center"
+        >
+          <h3 className="text-xl bg-gradient-to-r from-indigo-600 via-violet-500 to-blue-700 bg-clip-text text-transparent font-black">
+            100xJobs
+          </h3>{" "}
+        </MobileLink>
+        <div className="flex flex-col gap-3 mt-10 text-lg">
+          <MobileLink onOpenChange={setOpen} href="/">
+            Home
+          </MobileLink>
+          <MobileLink onOpenChange={setOpen} href="/jobs">
+            Explore
+          </MobileLink>
+          {userRole === "ADMIN" ? (
+            <Link href="/jobs/manage">
+              <p
+                className={cn("cursor-pointer hover:text-gray-900", {
+                  "text-gray-900": pathName === "/jobs/manage",
+                })}
+              >
+                Manage
+              </p>
+            </Link>
+          ) : null}
+
+          {!session ? (
+            <Link href="/login">
+              <Button className="font-medium">Join Now</Button>
+            </Link>
+          ) : (
+            <Button
+              onClick={handleSignOut}
+              variant="outline"
+              className="font-medium rounded-md"
+            >
+              Sign Out
+            </Button>
+          )}
+        </div>
+      </SheetContent>
+    </Sheet>
+  );
+};
+
+interface MobileLinkProps extends LinkProps {
+  children: React.ReactNode;
+  onOpenChange?: (open: boolean) => void;
+  className?: string;
+}
+
+function MobileLink({
+  href,
+  onOpenChange,
+  children,
+  className,
+  ...props
+}: MobileLinkProps) {
+  const router = useRouter();
+  return (
+    <Link
+      href={href}
+      onClick={() => {
+        router.push(href.toString());
+        onOpenChange?.(false);
+      }}
+      className={`text-black hover:text-green-600 transition duration-300 mb-5 ${className}`}
```

Comment:
why are you not using cn() here, makes the code more easily readable and clean + editable